spec/features/user_profile_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+feature 'user sees his and others profile', %Q{
+
+  As an authenticated user
+  I want to see my profile and others profile
+  So I can see my profile details and others.
+}do
+
+# Acceptance Criteria
+# [X] User clicking on "Profile" should show a page with his profile
+# [X] I should see my list of bots
+# [X] I should see a link to each bot show page
+# [X] I should see the the number of bots that I have
+# [ ] I should see my BAEs vocabulary (total of sentences, keywords and combination of keywords)
```

Comment:
Line is too long. [96/80]<br>Incorrect indentation detected (column 0 instead of 2).